File: Constructorio_Net/src/models/AutocompleteResponse.java
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

/**
 * Constructor.io Autocomplete Response
 */
namespace Constructorio_NET
{
    public class AutocompleteResponse
    {

        [JsonPropertyName("sections")]
        public Dictionary<String, List<Result>> Sections { get; set; }

        [JsonPropertyName("result_id")]
        public String ResultId { get; set; }

        [JsonPropertyName("request")]
        public Dictionary<String, Object> Request { get; set; }
    }
}

File: src/Constructorio_NET/models/Autocomplete/AutocompleteResponse.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

/**
 * Constructor.io Autocomplete Response
 */
namespace Constructorio_NET.Models
{
    public class AutocompleteResponse {

        [JsonProperty("sections")]
        public Dictionary<string, List<Result>> Sections { get; set; }

        [JsonProperty("result_id")]
        public string ResultId { get; set; }

        [JsonProperty("request")]
        public Dictionary<string, Object> Request { get; set; }
    }
}
